third_party/move/move-binary-format/src/errors.rs

```diff
@@ -309,10 +309,21 @@ impl PartialVMError {
     }
 
     pub fn new(major_status: StatusCode) -> Self {
+        debug_assert!(major_status != StatusCode::EXECUTED);
```

Comment:
we had typically not enabled these with the logic that we don't want different behavior from local and tests, but my memory isn't great either...

Reply:
Typically debug asserts are used to document preconditions, but okay to confirm to a aptos style guide if this is not accepted.